### PR TITLE
Add Riverflow

### PR DESCRIPTION
**Pull request by rinopo**
"Riverflow allows you to assign a two-finger gesture to an arbitrary action."

### CODE REVIEW

File: Casks/riverflow.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask :v1 => 'riverflow' do
+  version '1.0.0'
+  sha256 'b26f58eaa1abb58529a669aab6e9ad9d6a71780bf25a7c0a61d0684a3ebc4e93'
+
+  # amazonaws.com is the official download host per the vendor homepage
+  url 'https://s3-ap-northeast-1.amazonaws.com/sparkle-appcasts/riverflow/riverflow-1.0.0.zip'
```
Comment (adityadalal924) [inline Casks/riverflow.rb]: use double quotes and `#{version}` here instead of a hardcoded 1.0.0

File: Casks/configure-application-dock-tile.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  version '1.0'
+  sha256 '064f5c24ad7e26aa80c9c82b7c6049d96cd7396d53035fb4276fa638dc7d28c5'
+
+  url "http://boredzo.org/cadt/Configure_Application_Dock_Tile-#{version}.zip"
+  name 'Configure Application Dock Tile'
+  name 'CADT'
+  homepage 'http://boredzo.org/cadt/'
+  license :gratis
+
+  app 'Configure Application Dock Tile 1.0/Configure Application Dock Tile.app'
```
Comment (vitorgalvao) [inline Casks/configure-application-dock-tile.rb]: `app "Configure Application Dock Tile #{version}/Configure Application Dock Tile.app"`

**Comment by rinopo** [comment]
Sorry for the mess.

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in c6e386405f5d398efb60d9b15d5d10701ea52256. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**